### ISSUE TITLE
run-in-docker.sh script is broken

### ISSUE THREAD

**Issue by mthebridge** [swagger-api/swagger-codegen#8072]
##### Description

The `run-in-docker.sh` script fails with a docker syntax error.

##### Swagger-codegen version

Latest master branch (at f9ae84d)

##### Steps to reproduce

On latest master branch:
```bash
$ sudo ./run-in-docker.sh mvn package
++ dirname ./run-in-docker.sh
+ cd .
+ maven_cache_repo=/root/.m2/repository
+ mkdir -p /root/.m2/repository
++ id -u
++ id -g
+ docker run --rm -it -w /gen -e GEN_DIR=/gen -e MAVEN_CONFIG=/var/maven/.m2 -u 0:0 ' '
docker: invalid reference format.
See 'docker run --help'.
```
##### Related issues/PRs

Regression caused by e2c58fad - see [here](https://github.com/swagger-api/swagger-codegen/commit/e2c58fad717a9f802cc814b7dabc573fed78509c#diff-d38826a43246249bd290686af1eaa5e0)

##### Suggest a fix/enhancement
Remove the extra whitespace in the script.

**Comment by crhistianramirez** [comment]
Should this be closed then? #8070

**Comment by mthebridge** [comment]
Yes, sorry!
